### PR TITLE
UP-4783: Import Export Portlet - Add missing id input - master

### PR DESCRIPTION
**Pull request by ChristianMurphy**
https://issues.jasig.org/browse/UP-4783

#### Issue

Import Export portlet, delete view has an `<label>` that does not have an associated `<input>`

#### Resolution

Remove unused `<label>`

***edit***

id `<input>` and `<label>` are actually both needed. The `<input>` was accidentally removed before.

### CODE REVIEW

**Comment by ChristianMurphy** [comment]
Thanks @drewwills and @cousquer `id` input and label has been added to the delete view in the portlet.

screencapture-localhost-8080-uportal-p-importexportportlet-ctf3-max-render-up-1481817206244

File: uportal-war/src/main/webapp/WEB-INF/jsp/ImportExportPortlet/delete.jsp [hunk 1]
```diff
@@ -25,65 +25,96 @@
...
+            <spring:message code="delete.portlet.entities"/>
+        </h2>
         <div class="toolbar" role="toolbar">
             <ul class="btn-group">
-                <li class="btn"><a class="button btn btn-primary" href="<portlet:renderURL/>"><spring:message code="import"/> <i class="fa fa-upload"></i></a></li>
-                <li class="btn"><a class="button btn btn-primary" href="<portlet:renderURL><portlet:param name="action" value="export"/></portlet:renderURL>"><spring:message code="export"/> <i class="fa fa-download"></i></a></li>
+                <li class="btn">
+                    <a class="button btn btn-primary" href="<portlet:renderURL/>">
+                        <spring:message code="import"/>
+                        <i class="fa fa-upload"></i>
```
Comment (cousquer) [inline uportal-war/src/main/webapp/WEB-INF/jsp/ImportExportPortlet/delete.jsp]: `<i class="fa fa-upload" aria-hidden="true"></i>`

File: uportal-war/src/main/webapp/WEB-INF/jsp/ImportExportPortlet/delete.jsp [hunk 2]
```diff
@@ -25,65 +25,96 @@
...
+                    </a>
+                </li>
+                <li class="btn">
+                    <a class="button btn btn-primary" href="<portlet:renderURL><portlet:param name="action" value="export"/></portlet:renderURL>">
+                        <spring:message code="export"/>
+                        <i class="fa fa-download"></i>
```
Comment (cousquer) [inline uportal-war/src/main/webapp/WEB-INF/jsp/ImportExportPortlet/delete.jsp]: `<i class="fa fa-download" aria-hidden="true"></i>`

File: uportal-war/src/main/webapp/WEB-INF/jsp/ImportExportPortlet/delete.jsp [hunk 3]
```diff
@@ -25,65 +25,96 @@
...
-    <div class="fl-widget-content content portlet-content" role="main">   
-        
+    <div class="fl-widget-content content portlet-content" role="main">
+
         <!-- Messages -->
         <div class="portlet-msg-error portlet-msg error alert alert-danger" role="alert">
             <div class="titlebar">
-                <h3 class="title">Warning</h3>
+                <h3 class="title">
+                    Warning
```
Comment (cousquer) [inline uportal-war/src/main/webapp/WEB-INF/jsp/ImportExportPortlet/delete.jsp]: hard coded strings, 😄 
What about ?
:gb:
delete.portlet.warning=Warning
:fr:
delete.portlet.warning=Avertissement

File: uportal-war/src/main/webapp/WEB-INF/jsp/ImportExportPortlet/delete.jsp [hunk 4]
```diff
@@ -25,65 +25,96 @@
...
+                </h3>
             </div>
             <div class="content">
-                <p>Deleting some entities can do very bad things to your portal.  By default, all delete operations are disabled;  use this feature with caution.</p>
+                <p>
+                    Deleting some entities can do very bad things to your portal.  By default, all delete operations are disabled;  use this feature with caution.
```
Comment (cousquer) [inline uportal-war/src/main/webapp/WEB-INF/jsp/ImportExportPortlet/delete.jsp]: :gb:
delete.portlet.warning.explanations=Deleting some entities can do very bad things to your portal.  By default, all delete operations are disabled;  use this feature with caution.
:fr:
delete.portlet.warning.explanations=Supprimer des entit\u00E9s peut faire des choses tellement vilaines \u00E0 votre portail.  Par d\u00E9faut, toutes les op\u00E9rations de suppression sont d\u00E9sactiv\u00E9es;  utilisez cette fonctionnalit\u00E9 avec grande pr\u00E9caution.

File: uportal-war/src/main/webapp/WEB-INF/jsp/ImportExportPortlet/delete.jsp [hunk 5]
```diff
@@ -25,65 +25,96 @@
...
+                </p>
             </div>
         </div>
-        
+
         <!-- Note -->
         <div class="note" role="note">
-            <p>Select an entity to delete.</p>
+            <p>
+                Select an entity to delete.
```
Comment (cousquer) [inline uportal-war/src/main/webapp/WEB-INF/jsp/ImportExportPortlet/delete.jsp]: :gb:
delete.portlet.note.select.entitie=Select an entity to delete.
:fr:
delete.portlet.note.select.entitie=Selectionnez une entit\u00E9 \u00E0 supprimer.

File: uportal-war/src/main/webapp/WEB-INF/jsp/ImportExportPortlet/delete.jsp [hunk 6]
```diff
@@ -25,65 +25,96 @@
...
-                <label class="portlet-form-label" for="sysid"><spring:message code="id"/>:</label>
-                <button class="button btn primary" type="submit"><spring:message code="delete"/> <i class="fa fa-trash-o"></i></button>
+                <label class="portlet-form-label" for="${n}sysid">
+                    <spring:message code="id"/>:
+                </label>
+                &nbsp;
+                <input type="text" id="${n}sysid" class="form-control" name="sysid"/>
+                <button class="button btn primary" type="submit">
+                    <spring:message code="delete"/>
+                    <i class="fa fa-trash-o"></i>
```
Comment (cousquer) [inline uportal-war/src/main/webapp/WEB-INF/jsp/ImportExportPortlet/delete.jsp]: `<i class="fa fa-trash-o" aria-hidden="true"></i>`

**Comment by ChristianMurphy** [comment]
Thanks @cousquer code has been updated per review.
As well as a new danger icon flourish around warning message.

screencapture-localhost-8080-uportal-p-importexportportlet-ctf3-max-render-up-1481901039181

**Pull request merged**